scripts/functions/requirements/osx_brew

```diff
@@ -290,6 +311,8 @@ requirements_osx_brew_libs_default()
 
   requirements_osx_brew_check_custom homebrew/dupes # for: zlib
 
+  undesired_check openssl098
```

Comment:
Yeah, the `requirements_osx_brew_libs_default` is called in few places, that's why I put it there. Not sure if we should have a dedicated `uninstall` function. Possibly. Didn't want to invest too much of the time having RVM2 down the road...